src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantsController.kt

```diff
@@ -67,14 +74,32 @@ class ParticipantsController(
     return SimpleSuccessResponsePayload()
   }
 
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping
+  @Operation(summary = "List participants and their assigned projects.")
+  fun listParticipant(): ListParticipantResponsePayload {
+    val models = participantStore.findAll()
+    val details =
+        projectAcceleratorDetailsService.fetchParticipantProjectDetails().associateBy {
+          it.projectId
+        }
+
+    return ListParticipantResponsePayload(models.map { makeParticipantPayload(it, details) })
+  }
+
   @ApiResponse200
   @ApiResponse404
   @GetMapping("/{participantId}")
   @Operation(summary = "Gets information about a participant and its assigned projects.")
   fun getParticipant(@PathVariable participantId: ParticipantId): GetParticipantResponsePayload {
     val model = participantStore.fetchOneById(participantId)
+    val details =
+        projectAcceleratorDetailsService.fetchParticipantProjectDetails().associateBy {
+          it.projectId
+        }
```

Comment:
Similar comment to my one above: we could fetch just the projects in this specific participant rather than all of them.